_sass/layouts/_footer.scss

```diff
@@ -1,6 +1,6 @@
 footer {
   clear: left;
-  background-color: #414042;
+  // background-color: #414042;
```

Comment:
I'm fine with removing the line altogether, to keep it clean.

Reply:
I agree